### PR TITLE
Adds support for semi-automated system test execution on Composer

### LINKED ISSUE #83: Fix SubworkflowMapper: missing action_mapper and control_mapper parameters when invoked
https://polidea.slack.com/archives/CGH100JQP/p1555399187000200

### PR DESCRIPTION
**Pull request by potiuk**
Implements #53
Fixes #83

### CODE REVIEW

File: oozie-to-airflow/README.md [hunk 1]
```diff
@@ -315,3 +314,94 @@ To run all the tests in a given file call the below command:
...
+* primary disk size, 50 GB
+* Image 1.3.29-debian9
+* Hadoop version
+* Init action: TODO
+
+
+### Running the tests
+
+We can run examples defined in examples folder to run system test. The system tests use a real
+dataproc cluster, composer and oozie (in the future) to run the tests automatically.
```
Comment (sprzedwojski) [inline oozie-to-airflow/README.md]:
```suggestion
Dataproc cluster, Composer and Oozie (in the future) to run the tests automatically.
```

File: oozie-to-airflow/README.md [hunk 2]
```diff
@@ -315,3 +314,94 @@ To run all the tests in a given file call the below command:
...
+
+Each test consists of several phases:
+
+* convert - converts workflow from the example folder to DAG
```
Comment (sprzedwojski) [inline oozie-to-airflow/README.md]:
```suggestion
* convert - converts workflow from the example folder to an Airflow DAG
```
In Oozie they also sometimes call workflows DAGs, so maybe it's better to be specific.

File: oozie-to-airflow/README.md [hunk 3]
```diff
@@ -315,3 +314,94 @@ To run all the tests in a given file call the below command:
...
+* prepare-hadoop - prepares hadoop directory to run the tests (copises all necessary files)
```
Comment (sprzedwojski) [inline oozie-to-airflow/README.md]:
```suggestion
* prepare-hadoop - prepares a Hadoop directory to run the tests (copises all necessary files)
```

File: oozie-to-airflow/run-sys-test [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+then
+    usage
+fi
+
+
+eval set -- "${PARAMS}"
+unset PARAMS
+
+# Parse Flags.
+# Please update short and long options in the breeze-complete script
```
Comment (sprzedwojski) [inline oozie-to-airflow/run-sys-test]: `breeze-complete` Some copy pasting gone wrong? ;)

File: oozie-to-airflow/mappers/base_mapper.py [hunk 5]
```diff
@@ -91,3 +91,8 @@ def copy_extra_assets(self, input_directory_path: str, output_directory_path: st
         :return: None
         """
         return None
+
+    @staticmethod
+    def common_imports() -> Set[str]:
+        # TODO: for now, just in case we add all el functions as imports. We can optimize it later
```
Comment (mik-laj) [inline oozie-to-airflow/mappers/base_mapper.py]: I think, you should add this import to workflow
Please look at:
https://github.com/GoogleCloudPlatform/cloud-composer/blob/09569063ed46b5d2ec4ff9d50127b1f316619989/oozie-to-airflow/converter/primitives.py#L50-L56

Comment (potiuk) [reply oozie-to-airflow/mappers/base_mapper.py]: 👍  - did not know it exists :) Thanks!

File: oozie-to-airflow/run-sys-test (hunk 4 above)
Comment (potiuk) [reply oozie-to-airflow/run-sys-test]: 🗡

File: oozie-to-airflow/README.md [hunk 6]
```diff
@@ -315,3 +314,94 @@ To run all the tests in a given file call the below command:
...
+* n1-standard-2, 2 vCPU, 7.50 GB memory
+* primary disk size, 50 GB
+* Image 1.3.29-debian9
+* Hadoop version
+* Init action: TODO
+
+
+### Running the tests
+
+We can run examples defined in examples folder to run system test. The system tests use a real
```
[the remark this answers is not in the document]
Comment (potiuk) [reply oozie-to-airflow/README.md]: 👍

File: oozie-to-airflow/README.md (hunk 1 above)
Comment (potiuk) [reply oozie-to-airflow/README.md]: 👍

File: oozie-to-airflow/README.md (hunk 3 above)
Comment (potiuk) [reply oozie-to-airflow/README.md]: Already gone

**Pull request merged**